data/json/vehicleparts/boards.json

```diff
@@ -28,6 +30,7 @@
     "copy-from": "clothboard",
     "type": "vehicle_part",
     "name": "cloth quarterpanel",
+    "description": "A half-height cloth wall. Keeps zombies outside the vehiclebut allows people to see over it.",
```

Comment:
Missing a space here

Reply:
You fixed the double spacing, but I was actually talking about the typo ("vehiclebut").  I probably should have clarified after Night-Pryanik's comment.  Looks like the same typo got copy-pasta'd several times as well, including a few I missed last night